epoxy-processortest/build.gradle

```diff
@@ -11,6 +11,12 @@ android {
   defaultConfig {
     minSdkVersion rootProject.MIN_SDK_VERSION
     targetSdkVersion rootProject.TARGET_SDK_VERSION
+
+    javaCompileOptions {
+      annotationProcessorOptions {
+        includeCompileClasspath false
```

Comment:
This makes it so Gradle doesn't complain that there are undeclared annotation processors in the classpath.

Reply:
leave a code comment about it?

Reply:
That seems a bit redundant, the config itself is pretty explicit: don't include annotation processors that are in the compile classpath.

Reply:
well you still left a github comment explaining it, so it isn't _too_ explicit ;)

Reply:
Only because it doesn't seem directly related to this PR. 😛